src/index.html

```diff
@@ -22,6 +22,339 @@
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              href="#"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__block-card">
+        <article
+          data-qa="card"
+          class="main__card card"
+        >
+          <div class="card__img-block card__img-block--margin">
+            <img
+              src="images/imac.jpeg"
+              alt="Product photo"
+              class="card__product-photo"
+            />
+          </div>
+          <div class="card__description card__description--margin">
+            <p class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <p class="card__product-code">Product code: 195434</p>
+          </div>
+          <div class="card__rating stars">
+            <div class="stars__block stars__block--stars--4">
+              <div class="stars__star stars__star--1"></div>
+              <div class="stars__star stars__star--2"></div>
+              <div class="stars__star stars__star--3"></div>
+              <div class="stars__star stars__star--4"></div>
+              <div class="stars__star stars__star--5"></div>
+            </div>
+            <span class="stars__reviews">Reviews: 5</span>
+          </div>
+          <div class="card__sum card__sum--margin">
+            <span class="card__price">Price:</span>
+            <span class="card__cost">$2,199</span>
+          </div>
+          <button
+            data-qa="card-hover"
+            class="card__button"
+          >
+            Buy
+          </button>
+        </article>
+        <article class="main__card card">
+          <div class="card__img-block card__img-block--margin">
```

Comment:
Remove this class everywher
```suggestion
          <div class="card__img-block">
```